src/js/apps/forms/form/form_app.js

```diff
@@ -40,7 +40,12 @@ export default App.extend({
   initFormState() {
     const storedState = store.get(`form-state_${ this.currentUser.id }`);
 
-    this.setState(extend({ isActionSidebar: true, isExpanded: true, shouldShowHistory: false }, storedState));
+    this.setState(extend({
+      isActionSidebar: true,
+      isExpanded: true,
+      shouldShowHistory: false,
+      saveButtonType: 'save',
```

Comment:
This state value will either be `save` or `saveAndGoBack`. And determines whether the `Save Form` or `Save Form + Go Back` button is shown in the views.